site-src/implementations.md

```diff
@@ -17,7 +17,7 @@ Implementors and integrators of Gateway API are encouraged to update this docume
 - [Emissary-Ingress (Ambassador API Gateway)][4] (alpha)
 - [Envoy Gateway][18] (beta)
 - [Flomesh Service Mesh][17] (beta)
-- [Gloo Edge 2.0][5] (work in progress)
+- [Gloo Gateway 2.0][5] (beta)
```

Comment:
you can also add to the Gloo section to get a badge with something like this: https://github.com/kubernetes-sigs/gateway-api/blob/1ca2a9e3ffa51219e85aa3badde220cca7c3ad30/site-src/implementations.md?plain=1#L159